book/explore-patterns.md

```diff
@@ -0,0 +1,17 @@
+# Explore Patterns
+
+More and more patterns are contributed to this book by the InnerSource Commons community. That is awesome!
+
+Now how to make it easy for readers to discover the patterns that can help them in their particular situation?
+
+For this purpose we provide this mind map. It categorizes patterns based on the different phases of an InnerSource Program, and the challenges that might appear in the respective phases.
+
+<img src="./innersource-program-mind-map.png" title="InnerSource Patterns as a Mind Map">
+
+## Improve this Mind Map
+
+If you have other ideas for improving the discoverability of our patterns, or want to help us to improve this mind map, take a look at the documentation of our [Pattern Categorization](https://github.com/InnerSourceCommons/InnerSourcePatterns/blob/main/pattern-categorization/README.md) approach, and also check how to [contribute to this book](contribute.md).
+
```

Comment:
How about also adding the option of opening an issue if the readers noticing any problem that should be fixed? The rest looks good to me.

Reply:
Will work that in.

Reply:
Btw I took the "The rest looks good to me" as approval for this PR overall.